backend/src/auth/session_id.rs

```diff
@@ -69,9 +70,9 @@ impl SessionId {
 
     /// Returns a cookie for a `set-cookie` header in order to store the session
     /// ID in the client's cookie jar.
-    pub(crate) fn set_cookie(&self) -> Cookie {
-        // TODO: expiration and other cookie stuff!
-        Cookie::build(SESSION_COOKIE, base64encode(self.0.expose_secret()))
+    pub(crate) fn set_cookie(&self, max_age: Duration) -> Cookie {
+        // TODO: other cookie stuff!
```

Comment:
I think this TODO comment can be completely removed. The only thing that's missing is the `__Host` prefix and that's already tracked as an issue. (And even without the prefix, it's fine)